doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

```diff
@@ -53,6 +53,36 @@ Recommended changes
 
 The following changes are recommended for your application to work optimally after the migration.
 
+Build system
+============
+
+.. toggle::
+
+   * The default runner for the ``west flash`` command has been changed to use `nRF Util`_ instead of ``nrfjprog``, which was part of the archived `nRF Command Line Tools`_.
```

Comment:
```suggestion
   * The default runner for the ``west flash`` command has been changed to use `nRF Util`_ instead of ``nrfjprog`` that was part of the archived `nRF Command Line Tools`_.
```
is or was? You have is in the changelog.